findbugs-filter.xml

```diff
@@ -3,5 +3,8 @@
     <Match>
         <Class name="~com\.microservicesteam\.adele\..+\.domain\.Immutable.*" />
     </Match>
+    <Match>
+        <Class name="~com\.microservicesteam\.adele\..+\.model\.Immutable.*" />
```

Comment:
Maybe. Do you think it conveys additional value?
How come you don't need to exclude `events`?